primitives/staking-backport/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "sp-staking-backport"
```

Comment:
Do you see any possible issues with renaming it back to `sp-staking` after we reach the Substrate target version? If yes, can we somehow prevent them now?

Reply:
When the version from `polkadot-v0.9.36` branch reached, the `sp-staking-backport` crate should be removed and all references to `sp-staking-backport` should be replaced with `sp-staking`. I expect a quick and simple update.